src/mmrelay/matrix/credentials.py

```
@@ -0,0 +1,339 @@
+    CONFIG_KEY_DEVICE_ID,
+    CONFIG_KEY_HOMESERVER,
+    CONFIG_KEY_PASSWORD,
+    CONFIG_KEY_USER_ID,
+    REQUIRED_CREDENTIALS_KEYS,
+)
+
+__all__ = [
+    "_resolve_credentials_save_path",
+    "_missing_credentials_keys",
+    "_resolve_and_load_credentials",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)